src/meta-data/message-broker.py

```diff
@@ -156,6 +161,7 @@ def listen_to_created_sites():
                     "distance_to_nearest_trunk": distance_to_closest_trunk,
                     "distance_to_nearest_unclassified_road": distance_to_closest_unclassified_road,
                     "distance_to_nearest_motorway": distance_to_closest_motorway,
+                    "greenes_of_the_site":greenness_for_a_site,
```

Comment:
@wabinyai we could be having a typo here. "`greenes` or `greenness`?"

Reply:
typo greenness